content/10.zk-stack/20.running-a-zk-chain/80.validium.md

```diff
@@ -181,23 +181,155 @@ updates/optimizations or completely changing the DA verification logic, e.g. due
 <details>
 <summary>How to configure EigenDA client?</summary>
 
-1. Add the following block to the `general.yaml`, values used are only an example:
+## Common changes
+
+Changes needed both for local and mainnet/testnet setup.
+
+1. Add `da_client` to `etc/env/file_based/general.yaml`:
 
     ```yaml
     da_client:
       eigen:
-        rpc_node_url: https://disperser-holesky.eigenda.xyz:443
-        inclusion_polling_interval_ms: 2000
+        disperser_rpc: <your_desired_disperser>
+        settlement_layer_confirmation_depth: -1
+        eigenda_eth_rpc: <your_desired_rpc>
+        eigenda_svc_manager_address: '0xD4A7E1Bd8015057293f0D0A557088c286942e84b'
+        status_query_timeout: 1800000 # ms
+        status_query_interval: 5 # ms
+        wait_for_finalization: false
+        authenticated: false
+        verify_cert: true
+        path_to_points: ./resources
+        chain_id: <your_chain_id>
     ```
 
-2. Add the following block to the `secrets.yaml`:
+2. Also set the private key in `etc/env/file_based/secrets.yaml`:
 
     ```yaml
     da:
       eigen:
         private_key: YOUR_PRIVATE_KEY
     ```
 
+3. (optional) for using pubdata with 2MiB (as per specification), modify `etc/env/file_based/general.yaml`:
+
+    ```yaml
+    max_pubdata_per_batch: 2097152
```

Comment:
we actually have a limit of `761856` enforced in the system contracts
it should be removed soon, but still, let's not confuse the readers here